moderation/commands.go

```diff
@@ -778,6 +799,11 @@ func AdvancedDeleteMessages(channelID int64, filterUser int64, regex string, max
 			continue
 		}
 
+		// Check if pinned message to ignore ; no need to check if pinEnableFilter bool is true because if false , map will be empty.
+		if _, found := pinnedMessages[msgs[i].ID]; found {
```

Comment:
pretty sure this will panic if `pinFilterEnable` is false, since `pinnedMessages` will be nil

Reply:
Reading from empty maps dont cause panic iirc but writing would cause a runtime panic. Ill still wrap it up in an if statement.

Reply:
ah yeah you're right